De novo repeat annotation documnet.tex

```diff
@@ -1,321 +1,508 @@
+\newcommand\thickhline{\noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
+	\hline
+	\noalign{\global\arrayrulewidth\savedwidth}}
 
-\noindent * Analysis run on a slurm machine with 4$\sim$16 cpus, and 8GB RAM, running Red Hat Linux.\\
-\textcolor{red}{** New Bearded dragon analysis used same bearded dragon genome, except it was run on a High Performance Computing machine with 32 cpus, running Red Hat Linux.} \\
 
-\textbf{\textcolor{blue}{ \Large Finished! Good Luck!!!}}
+% Use adjustwidth environment to exceed column width (see example table in text)
+%\usepackage{changepage}
+\begin{document} \sloppy
+	
+	\begin{center}
+		\huge \textbf{CARP Document}
+	\end{center}
+	
+	% Insert table of contents
+	\tableofcontents
+	\newpage
+	
+	\pagebreak
+	
+	%------------------------------------------------------------%
+	%------------------------- Biogo Document -------------------%
+	%------------------------------------------------------------%
+	
+	
+	% CARP documentation
+	\begin{center}
+		\section{CARP manual 1.1}
+		\noindent\textbf{b\'iogo written by: Dan Kortschak} \\
+		\textbf{Document organised by: Lu Zeng} \\
+		\today
+	\end{center}
+	
+	\noindent krishna and igor are \textit{ab initio} repeat family identification and annotation programs, that  identify repeat element boundaries and family relationships from whole-genome sequence data. These programs build, refine and classify consensus models of putative interspersed repeats. krishna and igor are built using b\'iogo \href{<url>}(\url{https://github.com/biogo/biogo/}), a bioinformatics library for the Go language. 
+	
+	\noindent \textbf{Disclaimer} This document is provided to assist researchers with linux command line experience.  We have done our best to provide usable instructions, examples and advice, but users assume full responsibility for the output they generate and the authors accept no responsibility for user generated output from any programs or methods listed herein.  
+	
+	%------------------------------------------------------------------
+	% Load prerequisite packages 
+	%------------------------------------------------------------------
+	
+	\subsection{{Prerequisites}}
+	
+	\subsubsection{ Download Go }
+	Available at \href{<url>} (\url{https://golang.org/dl}).\\
+	For installation details, follow the instructions on the \href{https://golang.org/doc/install#install} {\color{blue}Go installation} page. 
+	
+	\subsubsection{Git}
+	To perform the next step you will need Git to be installed. (Check that you have a git command before proceeding.) \\
+	
+	\noindent If you do not have a working Git installation, follow the instructions on the \href{https://git-scm.com/downloads}{\color{blue}Git download} page.
+	
+	\subsubsection{Download b\'iogo Packages}
+	{\color{red} Note:}  For convenience, add the workspace's bin subdirectory to your PATH, or add in \$HOME/.profile. 
+	\begin{lstlisting}[language=bash]
+	export PATH=$PATH:$(go env GOPATH)/bin
```

Comment:
I suspect that the `\end{lstlisting}` being one line down is what's contributing to the unnecessary blank line in the PDF, can you try making these so that the last line of quoted code is terminated with the `\end{lstlisting}` tag. If it fixes the problem, apply throughout.

Reply:
done throughout